clients/baseclient/base_client.go

```diff
@@ -25,11 +26,12 @@ func (c *BaseClient) SetTokenFactory(tokenFactory TokenFactory) {
 }
 
 // NewHTTPTransport creates a new HTTP transport
-func NewHTTPTransport(host, url, encodedUrl string, rt http.RoundTripper, jar http.CookieJar) *client.Runtime {
+func NewHTTPTransport(host, url, encodedUrl string, rt http.RoundTripper) *client.Runtime {
 	// TODO: apply the changes made by Boyan here, as after the update of the dependencies the changes are not available
 	transport := client.New(host, url, encodedUrl, schemes)
 	transport.Consumers["text/html"] = runtime.TextConsumer()
 	transport.Transport = rt
+	jar, _ := cookiejar.New(nil)
```

Comment:
what is this throws an error and doesn't manage to create the new cookie jar?

Reply:
The implementation never throws an error, I don't know why it has the second return parameter.